src/components/inputs/password-input.tsx

```diff
@@ -0,0 +1,64 @@
+import { Icon } from 'native-base';
```

Comment:
Is it possible not to take dependency on third party library?

Reply:
Unresolving this.

Ideally, would like to avoid using third party lib for basic components.